clear.lua

```diff
@@ -9,10 +9,8 @@ function HandleClearCommand(Split, Player)
     if Player:HasPermission("core.admin.clear") then
         local InventoryCleared = false;
         local ClearInventory = function(OtherPlayer)
-            if (OtherPlayer:GetName() == Split[2]) then
-                OtherPlayer:GetInventory():Clear()
-                InventoryCleared = true
-            end
+            OtherPlayer:GetInventory():Clear()
+            InventoryCleared = true
```

Comment:
This is not so good. The `FindAndDoWithPlayer` function actually calls the callback for the nearest name, it doesn't have to match.
Consider there are three players, Admin, Xoft and Xof. Now admin wants to cleat Xof's inventory, so he types `/clear Xof`. Just before sending the command, Xof disconnects, and Xoft's inventory gets cleared.